orbit/cmd/desktop/desktop.go

```diff
@@ -341,7 +349,15 @@ func main() {
 				}
 				myDeviceItem.Enable()
 
-				shouldRunMigrator := sum.Notifications.NeedsMDMMigration || sum.Notifications.RenewEnrollmentProfile
+				// Check our file to see if we should migrate
+				migrationInProgress, err := mdmMigrator.MigrationInProgress()
+				if err != nil {
+					log.Debug().Err(err).Msg("JVE_LOG: checking if migration file exists at top of loop")
```

Comment:
just a reminder for when you get to this, for all the migration errors, we not only want to log/return them but we also want to report them to the Fleet server.